scene/3d/mesh_instance_3d.cpp

```diff
@@ -332,8 +332,8 @@ void MeshInstance3D::create_multiple_convex_collisions(const Ref<MeshConvexDecom
 
 void MeshInstance3D::_notification(int p_what) {
 	switch (p_what) {
-		case NOTIFICATION_ENTER_TREE: {
-			_resolve_skeleton_path();
+		case NOTIFICATION_READY: {
+			call_deferred("_resolve_skeleton_path");
```

Comment:
Any reason this can't be a normal deferred call?

Reply:
I have confirmed that normal ready did not solve the MRP issue, I expect it is because Skeleton always uses the deferred process for Skin updates, but as I mentioned in the overview, I don't know if this is the right direction for a fix.

Reply:
No I meant:
```suggestion
	callable_mp(this, &MeshInstance3D::_resolve_skeleton_path).call_deferred();
```

Reply:
I think she meant you could use `callable_mp`, and then you don't have to bind the method.
```
callable_mp(this, &MeshInstance3D::_resolve_skeleton_path).call_deferred();
```

(it may need to be `public` in C++ though, not sure)

Reply:
Ah do you means it should be used `callable_mp().call_deferred()` ? Yes, that's true. I'll rewrite it.